src/main/java/no/nav/pensjonbrevdata/BrevdataEndpoint.java

```diff
@@ -64,11 +65,11 @@ public Brevdata getBrevForBrevkode(@PathVariable(value = "brevkode") String brev
     }
 
     @GetMapping("/brevdataForSaktype/{saktype}")
-    public List<Brevdata> getBrevdataForSaktype(@PathVariable(value = "saktype") String saktype, @RequestParam(value = "includeXsd") boolean includeXsd) {
+    public List<BrevdataDTO> getBrevdataForSaktype(@PathVariable(value = "saktype") String saktype, @RequestParam(value = "includeXsd") boolean includeXsd) {
         try {
             List<Brevdata> brevdata = provider.getBrevdataForSaktype(saktype);
-            return includeXsd ? brevdata.stream().map((brev) -> brev.medXSD(dokumentmalGenerator, fellesmalGenerator)).collect(Collectors.toList()) :
-                    brevdata;
+            return (includeXsd ? brevdata.stream().map((brev) -> brev.medXSD(dokumentmalGenerator, fellesmalGenerator)).collect(Collectors.toList()) :
```

Comment:
bare for konsistens kanskje også fint å bruke Collectors.toUnmodifiableList() i stedet for Collectors.toList()